susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -61,6 +61,13 @@ def _get_ac_storage_filenamepath():
 def check_reboot_required(target_sls):
     sls_file_on_minion = __salt__['cp.cache_file']('{0}{1}.sls'.format('salt://actionchains/', target_sls.replace('actionchains.','')))
     current_state_info = _read_sls_file(sls_file_on_minion)
+    if current_state_info is None:
+        return False
+    if not 'schedule_next_chunk' in current_state_info:
+        return False
+    if not 'mgrcompat.module_run' in current_state_info['schedule_next_chunk']:
+        return False
```

Comment:
Could you please describe in a code comment here, or maybe provide some documentation for this function to explain the reasoning behind of these different cases you are taking care here?

It is not really clear to me in which cases we can have the file existing but not containing the expected data, so it would be great to have some documentation for this function to understand the expectations.

Thanks!

Reply:
ok, it took me a while to remember all the cases, but now it should look better :) . Thanks for the suggestion!